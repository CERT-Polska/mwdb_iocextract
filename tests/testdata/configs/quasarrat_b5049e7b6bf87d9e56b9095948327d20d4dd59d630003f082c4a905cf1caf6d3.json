{"encryption_key": "JSnet_Juju", "hosts": ["nasjshome.myqnapcloud.com:2411"], "install_name": "JSnet.exe", "log_directory": "Logs", "mutex": "JJHu8jZ3jUQZzBkGSg", "startup_key": "JSnet", "subdirectory": "JSnet", "type": "quasarrat", "version": "1.4.0.0"}