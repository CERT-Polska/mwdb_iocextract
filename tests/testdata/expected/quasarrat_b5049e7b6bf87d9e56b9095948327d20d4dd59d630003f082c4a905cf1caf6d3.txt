Password JSnet_Juju
NetLoc nasjshome.myqnapcloud.com:2411/
Mutex JJHu8jZ3jUQZzBkGSg
Drop JSnet.exe
